packages/renderer/src/AppNavigation.svelte

```diff
@@ -37,28 +48,45 @@ function handleClick(): void {
     handleNavigation({ page: NavigationPage.RESOURCES });
   }
 }
+
+function onDidChangeConfigurationCallback(e: Event): void {
+  if ('detail' in e) {
+    const detail = e.detail as { key: string; value: string };
+    if (NAV_BAR_LAYOUT === detail?.key) {
+      iconWithTitle = detail.value === 'icon + title';
+    }
+    console.log(detail.value);
+  }
+}
 </script>
 
 <svelte:window />
 <nav
   class="group w-leftnavbar min-w-leftnavbar flex flex-col hover:overflow-y-none bg-[var(--pd-global-nav-bg)] border-[var(--pd-global-nav-bg-border)] border-r-[1px]"
   aria-label="AppNavigation">
-  <NavItem href="/" tooltip="Dashboard" bind:meta={meta}>
+  <NavItem href="/" tooltip="Dashboard" bind:meta={meta} bind:iconWithTitle={iconWithTitle}>
```

Comment:
Is `bind:` necessary here? NavItem is not changing its value, a `iconWithTitle={iconWithTitle}` would be enough I think